### ISSUE TITLE
operator position

### ISSUE THREAD

**Issue by bwgjoseph** [bwgjoseph/mongoose-vs-ottoman#86]
Hi,

On top of what was mentioned in https://github.com/bwgjoseph/mongoose-vs-ottoman/issues/53#issuecomment-849603597, I think in general, there isn't a place to look at the example of all query operator using `ottoman` API. All those mentioned in [query-builder](https://v2.ottomanjs.com/guides/query-builder.html#query-builder) aren't of much use when not using `query`.

Some of my confusion now is that when should an operator be in front of the field and when should it not be.

For example

```js
// when using $eq
// <field> then <operator>
.find({
	name : {
		$eq : 'Couchbase Airlines'
	}
}

// when using $in
// <operator> then <field>
.find({
	$in: {
		search_expr: 'model',
		target_expr: ['767-300F']
		}
}
```

Any way to properly identify where it should be defined?

Thanks

**Comment by AV25242** [comment]
Have crated a documentation investigation issue

**Comment by bwgjoseph** [comment]
This should probably be closed with #87
